mediaqueries-4/Overview.bs

```diff
@@ -1695,6 +1695,159 @@ Color Display Quality: the 'color-gamut' feature</h3>
+	Value: standard | high
+	For: @media
+	Type: discrete
+	</pre>
+</h3>
+
+High dynamic range (HDR) represents the combination of brightness, color depth, transfer function, and color gamut.
+
+<dl dfn-type=value dfn-for="dynamic-range">
+	<dt><dfn>high</dfn>
+	<dd>
+		If the following are true for the UA and the output device:
+		* max brightness is considered high
+		* color depth is greater than 24 bit
+		* has a <a href="#contrast-of-display">high contrast ratio</a>
+	<dt><dfn>standard</dfn>
+	<dd>
+		If the following are true for the UA and the output device:
+		* max brightness is considered standards
+		* color depth is 24 bit or 8 bit per color component of RGB
+		* has a <a href="#contrast-of-display">low contrast ratio</a>
+</dl>
+
+<h3 id="contrast-brightness-of-display">
+	Determining contrast and brightness of display</h3>
+
+This text is non-normative. There currently is no agreed upon standardization of
+high contrast ratio, low contrast ratio, max brightness or low brightness as it
+relates to displays. As such, the determination for <a>dynamic-range</a> and
+<a>video-dymanic-range</a> will be defined by the user agent.
+
+<h3 id="video-prefixed-properties">
+Handling bi-plane scenarios</h3>
+
+	In some scenarios, such as TVs, the user agents may have a graphics
+	plane dedicated to video that has its own discrete hardware. In order
+	to allow UAs in environments that have more than one plane for graphics
+	rendering the following properties should be used to correctly answer
+	video specific queries.
+
+	For any UA on a bi-plane device you **MUST** return the values for:
+	<a>video-color-gamut</a>; <a>video-width</a>; <a>video-height</a>; <a>video-resolution</a>;
+	<a>video-dynamic-range</a>; based on the plane that renders video. If the UA is not in a bi-plane
+	scenario then it **MUST** return the values of the single plane
+	(they're effectively aliases to their non-video counterparts).
+
+<h3 id="video-color-gamut">
+Video Color Display Quality: the 'video-color-gamut' feature</h3>
+
+	<pre class='descdef mq'>
+	Name: video-color-gamut
+	Value: srgb | p3 | rec2020
+	For: @media
+	Type: discrete
+	</pre>
+</h3>
+
+The 'video-color-gamut' media feature describes the approximate range of colors
+	that are supported by the UA and output device's video plane.
+	That is, if the UA receives content with colors in the specified space
+	it can cause the output device to render the appropriate color,
+	or something appropriately close enough.
+
+Value and color space definitions are the same as <a href="#color-gamut">color-gamut</a>
+
+<h3 id="video-dynamic-range">
+Video Dynamic Range: the 'video-dynamic-range' feature</h3>
+
+	<pre class='descdef mq'>
+	Name: video-dynamic-range
+	Value: standard | high
+	For: @media
+	Type: discrete
+	</pre>
+</h3>
+
+High dynamic range (HDR) represents the combination of brightness, color depth, transfer function, and color gamut.
+
+<dl dfn-type=value dfn-for="video-dynamic-range">
```

Comment:
I'd make the same edits here as for dynamic-range. But you might instead give a pointer to the non-prefixed feature like for video-color-gamut. ex: "Dynamic range values and definitions are the same as <a href="#dynamic-range">dynamic-range</a>"

Reply:
If you take that second option, be sure to add "by the UA and output device's video plane." to the end of the intro sentence on l.1779.

Reply:
Now that you're referencing the values for non-prefixed dynamic-range, you can remove this whole dfn block of value definitions and just rely on the reference. This is consistent with video-color-gamut.